modules/utils/src/merkle.ts

```diff
@@ -1,26 +1,40 @@
+import * as merkle from "@graphprotocol/vector-merkle-tree";
 import { CoreTransferState } from "@connext/vector-types";
 import { HashZero } from "@ethersproject/constants";
 import { keccak256 } from "ethereumjs-util";
 import { MerkleTree } from "merkletreejs";
 
-import { hashCoreTransferState } from "./transfers";
-
-export const generateMerkleTreeData = (transfers: CoreTransferState[]): { root: string; tree: MerkleTree } => {
-  // Sort transfers alphabetically by id
-  const sorted = transfers.sort((a, b) => a.transferId.localeCompare(b.transferId));
+import { encodeCoreTransferState, hashCoreTransferState } from "./transfers";
 
+export const generateMerkleTreeData = (transfers: CoreTransferState[]): { root: string; tree: merkle.Tree } => {
```

Comment:
Looks like that previous comment about not returning an unusable `merkle.Tree` (that has been `free`'d) has not yet been implemented.